spec/models/miq_widget_spec.rb

```diff
@@ -318,6 +318,10 @@ def add_dashboard_for_user(db_name, userid, group)
       }
     end
 
+    it "returns MiqTask id if successful" do
+      expect(@widget.queue_generate_content).to equal(MiqTask.last.id)
```

Comment:
not sure whether `MiqTask.last` can work always(some other test can recreate new record of `MiqTask` ?). what about  to check if result of this query is present?

`MiqTask.where(:name => "Generate Widget: '#{@widget.title}'", :miq_task_id => @widget.reload.miq_task_id )`

thanks!

Reply:
@lpichler Fixed as suggested :)